mlrun/api/utils/scheduler.py

```diff
@@ -120,6 +120,17 @@ def create_schedule(
             concurrency_limit,
             auth_info,
         )
+        job_id = self._resolve_job_id(project, name)
+        job = self._scheduler.get_job(job_id)
+        if job and job.next_run_time:
+            logger.info(
+                "updating schedule with next_run_time",
+                job=job,
+                next_run_time=job.next_run_time,
+            )
+            get_db().update_schedule(
+                db_session, project, name, next_run_time=job.next_run_time
+            )
```

Comment:
It's a bit inefficient - a few lines before you called `create_schedule` which doesn't get a `next_run_time` parameter. So, you create the schedule in the DB, then immediately after call `update_schedule` instead of creating the schedule in the scheduler, getting its `next_run_time` and creating the DB schedule with the `next_run_time` already updated. 
Not terrible (and there are advantages to first storing in the DB, only then updating in the local scheduler), but something to consider.